api_mt/integrations/api_keys.md

```diff
@@ -0,0 +1,141 @@
+# How to obtain API keys for MT plugins  
```

Comment:
There could be many reasons that someone wants to get an API key, not just for a plugin.

We make this page more popular, we should avoid making it specifically about "plugins" or even "CAT".  (The actual content you wrote is not specific to that.)

Instead, we can just briefly explain why one would need to get an API key.

> A machine translation **API key** is required to integrate machine translation into other applications and systems, either as a developer or as a user.
> - to integrate a machine translation API into systems and applications
> - use a machine translation API integration in a translation application, like a TMS, CAT tool or CAT tool plugin

(I don't love my wording here, just an example.)

Also:
- I generally don't like "obtain" - too fancy ;-)
- We can't use "MT" or other abbreviations.

Reply:
I updated title and added introduction as @bittlingmayer proposed. I used "get" instead of "obtain" :)